pkg/frontend/v2/frontend_test.go

```diff
@@ -3,12 +3,15 @@ package v2
 import (
 	"context"
 	"net"
+	"net/url"
 	"strconv"
 	"strings"
 	"sync"
 	"testing"
 	"time"
 
+	"github.com/cortexproject/cortex/pkg/scheduler/queue"
```

Comment:
Small nit. Let's move this line to the section of L22

Reply:
Addressed in https://github.com/cortexproject/cortex/pull/5605/commits/96ff019ddba591815be280b4f9c9aacf63733a42